tests/components/google_assistant/test_smart_home.py

```diff
@@ -598,6 +600,89 @@ def registries(hass):
     }
 
 
+@pytest.mark.parametrize("device_class,google_type", [
+    ('door', 'action.devices.types.DOOR'),
+    ('garage_door', 'action.devices.types.SENSOR'),
```

Comment:
This seems incorrect, as we have this map to `TYPE_GARAGE` ?

Reply:
Not currently for the binary sensor only for cover.